bower.json

```diff
@@ -71,7 +71,7 @@
   "devDependencies": {
     "chai": "3.3.0",
     "mocha": "2.3.3",
-    "sinon": "http://sinonjs.org/releases/sinon-1.17.0.js",
+    "sinon": "http://sinonjs.org/releases/sinon-1.17.7.js",
```

Comment:
server vs. browser js libs. sinon gets loaded into the browser where as npm is for server js like cli calls to the js-xpath module